### ISSUE TITLE
Unexpected black edge after finetuning

### ISSUE THREAD

**Issue by cyysc1998** [alimama-creative/FLUX-Controlnet-Inpainting#38]
Thanks for sharing the excellent work! I finetune the model on my own dataset about 3k step with lr 5e-6. The input of the controlnet is the concatenation of normalized masked_image and binarized mask.  I find there are some black edge at the edge of the brushed area. The boundary is very obvious. Can you give me some hint?
![image](https://github.com/user-attachments/assets/a039a241-7ce2-48e5-8d24-b1797c3ad499)

**Comment by ykj467422034** [comment]
> Thanks for sharing the excellent work! I finetune the model on my own dataset about 3k step with lr 5e-6. The input of the controlnet is the concatenation of normalized masked_image and binarized mask. I find there are some black edge at the edge of the brushed area. The boundary is very obvious. Can you give me some hint?感谢您分享出色的作品！我使用 lr 5e-6 在自己的数据集上对模型进行了大约 3k 步的微调。控制网的输入是归一化 masked_image 和二值化 mask 的串联。我发现拉丝区域的边缘有一些黑边。界限非常明显。你能给我一些提示吗？ ![image](https://private-user-images.githubusercontent.com/35159700/387168508-a039a241-7ce2-48e5-8d24-b1797c3ad499.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI3ODQ4OTIsIm5iZiI6MTczMjc4NDU5MiwicGF0aCI6Ii8zNTE1OTcwMC8zODcxNjg1MDgtYTAzOWEyNDEtN2NlMi00OGU1LThkMjQtYjE3OTdjM2FkNDk5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTI4VDA5MDMxMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBjNjMxNzRhMWJhZDU5ZmE1ODY1NzBhYzY2ZDNhMTJhOTQxNDg1NzQwYWRjZDQ4NGFkZWU0ODFmODYzYzcyOGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.GRhzRReDr62-iaKBgZSa84eGy0uaEpvEMj0UwkluGH4)

Hello，I am preparing to fine-tune this model, but I don’t have the training code. Can you share it for learning? Thanks！ @cyysc1998
